elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -297,6 +297,9 @@
     @Message(id = 35, value = "Unable to load module '%s'.")
     OperationFailedException unableToLoadModule(String module, @Cause Exception cause);
 
+    @Message(id = 1211, value = "KeyStore does not contain a PublicKey and PrivateKey for KeyStore: [%s] and alias: [%s].")
```

Comment:
detail but we could place this message below just because of its `id`

Reply:
Added